mne/source_estimate.py

```diff
@@ -1041,8 +1017,8 @@ def __repr__(self):  # noqa: D105
         s += ", tmin : %s (ms)" % (1e3 * self.tmin)
         s += ", tmax : %s (ms)" % (1e3 * self.times[-1])
         s += ", tstep : %s (ms)" % (1e3 * self.tstep)
-        s += ", data size : %s x %s" % self.shape
-        return "<SourceEstimate  |  %s>" % s
+        s += ", data shape : %s" % (self.shape,)
+        return "<%s  |  %s>" % (type(self).__name__, s)
```

Comment:
I think I changed it from `__class__` to `type(self)` at some point at the request of someone... can't remember.